hello-thirdparty/README.md

```diff
@@ -5,13 +5,16 @@ Hello Third Party Lib is an Android C++ sample that depends on a prebuilt static
 Pre-requisites
 --------------
 - Android Studio 1.3+ with [NDK](https://developer.android.com/ndk/) bundle.
-- [Play Games C++ SDK](https://developers.google.com/games/services/downloads/sdks)
+
+Project structure
```

Comment:
I had it towards the end first, but I thought since it is downloading a zip file and including code it should be disclosed before the building steps?  I can move if you think it is better